clang/lib/Driver/ToolChains/PS4CPU.cpp

```diff
@@ -295,6 +302,9 @@ void tools::PS5cpu::Linker::ConstructJob(Compilation &C, const JobAction &JA,
   if (Shared)
     CmdArgs.push_back("--shared");
 
+  if (!Relocatable && !Shared && !PIE)
```

Comment:
Does this want to consider `Static` as well? Just wondering, no opinion either way.

Reply:
The intention is to supply a base for non-PIE executables, which by default includes `-static`. I'll add a comment to ensure it doesn't appear accidental.

We have a distinct linker script for `-static` executables (which isn't visible here just yet, but soon). That script currently hardcodes a base address, but with this change it will be possible to remove that and allow the base to be overridden from the command line, while still having the same default.